wp-content/themes/pub/wporg-learn-2020/css/components/_page-lesson-plan.scss

```
@@ -0,0 +1,137 @@
+.lesson-plan {
```

The convention within this repo for this prefix is `lp` from what I can see
```suggestion
.lp {
```